### DISCUSSION TITLE
GPG -error NOPUPKEY

### DISCUSSION THREAD

**Discussion by OSallavan** [MiczFlor/RPi-Jukebox-RFID#1811] [Q&A] [answered]
Hello Guys

i have a Problem with the PUBKEY (mobidy)

i have used the Image from MiczFlor	
OS Distribution
Raspbian
Description
Raspbian GNU/Linux 10 (buster)
Release
10
Codename
buster

	
Version
2.3 - 24258901 - master
Edition
Classic edition (barebones)
MPD Server Status

i have installed it on a SD-Card to testing and i have autedication Sptify with mobidy.
Now  I want Phonibox on a bigger SD-Card and now i get the GPG-Error

Warning: apt-key output should not be parsed (stdout is not a terminal)
OK
W: GPG error: https://apt.mopidy.com/ buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY <PUPKEY>

What is the Problem?
Who can help me?
If you need more info´s write please.

Thank U for your help.

best regards
O

**Comment by s-martin** [comment, marked as answer]
Is fixed here https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1792

but this is not part of the latest release.

**Reply by OSallavan** [reply]
Thank U for yur answer :)
Now it works, i dont have the Problem with den PubKey. 
I had access denied in Spotify.
And than doing your answer.

The next Problem, the RFID-Reader dosen´t work.
Like this
https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/747
 sudo pip3 install evdev
cd /home/pi/RPi-Jukebox-RFID/scripts/
python3 RegisterDevice.py

helps.

It´s work´s.
(Lokal and Spotify-Playlist)
Im reboot the raspi an spotify dosent work.
Song in the Card work.

i look in this 
journalctl -u mopidy

here i can see - UNABLED_TO_CONECT_SERVER

But mobidy works - status ist Connected - i see it in the WebApp

Is that the Problem from the Phonibox or from Spotify?

Thank u for your help.

Best Regards
O.

**Reply by s-martin** [reply]
And that’s this bug
https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1791

Also not yet in latest release

**Reply by OSallavan** [reply]
Hallo,

thank U for your answer :)

now i have the problem with the on/off shim
i doing this
curl https://get.pimoroni.com/onoffshim | bash
and i get this
curl:(6) could not resolve host get.pimoroni.com

i don´t know.. why i have so many problems?

What i doing wrong?

best regards
O.

**Reply by s-martin** [reply]
The problem with on off shim seems to b a lack of connection to the pimoroni website.
Maybe only temporary?

If this is still an issue, please open a new discussion.
